rfcs/interop_charter.md

```diff
@@ -0,0 +1,17 @@
+# RFC 120: Interop Team Charter
+
+## Summary
+
+An Interop Team is created as part of the web-platform-tests project, defined by a charter [in the team repo](https://github.com/web-platform-tests/interop-2022/pull/102). The team is tasked with maintaining [Interop 2022](./interop_2022.md), [Interop 2023](./interop_2023.md) and future such efforts.
+
+## Details
+
+The Interop 2022 [repo](https://github.com/web-platform-tests/interop-2022) and [team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will be renamed to omit the year, as described in the [Interop 2023 RFC](./interop_2023.md).
+
+The [team charter](https://github.com/web-platform-tests/interop-2022/pull/102) is made official by this RFC and substantial changes can only be made through the web-platform-tests RFC process.
+
+The Interop Team can define additional public efforts/metrics beyond Interop 2022 and 2023. The process for doing that is decided by the Interop Team, may evolve over time, and is not defined by this RFC.
```

Comment:
This kinda implies Interop could create a new BSF, which begs the question of "should Interop own BSF".

Reply:
Yes, I think it does imply we could create that metric, if it didn't exist already. Do you think we should explicitly give the team the authority to change or remove the BSF metric on wpt.fyi, or how do you answer the begged question?

Reply:
I don't have a particularly strong opinion, but am mildly in favour of doing so, I think.

Reply:
I've added this language:

> The Interop Team also maintains the "Browser Specific Failures" metric on [wpt.fyi](https://wpt.fyi/).

Reply:
It's also mentioned in https://github.com/web-platform-tests/interop/pull/102 now.